src/middleware/middleware.js

```diff
@@ -0,0 +1,33 @@
+export const validateEmail = (email) => {
+  // eslint-disable-next-line no-useless-escape
+  const mailformat = /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/;
+  return mailformat.test(email);
+};
+export const validPhoneNo = (number) => {
+  const phoneno = /^\d{10}$/;
```

Comment:
is this the phone number or the `phoneNumberRegex` ? 👀

Reply:
phoneNumberRegex